portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -27,22 +27,34 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-    String convertTojson(ArrayList<String> comments) {
-        Gson gson = new Gson();
-        String convertedTojson = gson.toJson(comments);
-        return convertedTojson;
-    }
+  private String comments = "";
+  private ArrayList<String> messages = new ArrayList<>();
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    ArrayList<String> messages = new ArrayList<>();
 
-    messages.add(0, "Star Wars is awesome!");
-    messages.add(1, "I will become the next UN secretary general.");
-    messages.add(2, "I am to go back to school.");
-
-    String comments = messages.get((int) (Math.random() * messages.size()));
     response.setContentType("text/html;");
     response.getWriter().println(comments);
   }
+  /*
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+      String statement = comment(request, "texts");
+      messages.add(statement);
+
+      if (messages.size() != 101) {
```

Comment:
I would add a comment here - what is special about the 'magic number' 101? Any time you have a magic number (ie a hard coded value that is there for some reason that is not immediately apparent) you probably want to do a comment. Additionally it can be useful to pull these kinds of numbers out to constants or global variables. 

Looking at this code it looks like you are doing some formatting / wrapping. That makes a lot of sense but imagine if you need to change this from 101 to 201? Now you have to find every piece of code where you used that value. It's much easier to do something like INT TOTAL_LENGTH=101; and use that everywhere you need to use that magic number. This makes refactoring (and testing!) much easier in most cases.

Reply:
101 is not a constant or anything special. I wanted to get back to it later and work out the proper number for the function to know when it should rewrite the comment string